### DISCUSSION TITLE
Is there a reason why BT-759 received submission count and BT-760 Received Submissions Type are removed from the 30.efx?

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#507] [Q&A] [answered]
Is there a reason why BT-759 received submission count and BT-760 Received Submissions Type are removed from the 30.efx? 
They are not in de form 30 efx. But tjey are in the form 29 efx.
They should show in section 6.4 of the publication.

**30 EFX** 

		4 {OPT-322-LotResult[(BT-712(a)-LotResult is present) or (BT-636-LotResult is present) or (BT-760-LotResult is present) or (BT-710-LotResult is present) or (BT-710-LotResult is present) or (BT-735-LotResult is present)]} #{auxiliary|text|statistical-information} // 6.1.4. Statistical information
			0 {OPT-322-LotResult[BT-712(a)-LotResult is present]} #{auxiliary|text|review-requests-summary} // Summary of the review requests the buyer received
				0 {BT-712(a)-LotResult} #{name|BT-712}: ${BT-712(b)-LotResult} // Buyer Review Complainants
					{BT-197(BT-712)-LotResult} #{name|BT-197(BT-712)-LotResult}: #{BT-197(BT-712)-LotResult} // Unpublished Justification Code
					{BT-196(BT-712)-LotResult} #{name|BT-196(BT-712)-LotResult}: ${BT-196(BT-712)-LotResult} // Unpublished Justification Description
					{BT-198(BT-712)-LotResult} #{name|BT-198(BT-712)-LotResult}: ${BT-198(BT-712)-LotResult} // Unpublished Access Date
			0 {OPT-322-LotResult} #{auxiliary|text|review-requests-irregularity-type} // Type of alleged irregularity
			0 {OPT-322-LotResult[BT-760-LotResult is present]} #{auxiliary|text|received-submissions} // Received tenders or requests to participate
			0 {OPT-322-LotResult[(BT-710-LotResult is present) or (BT-710-LotResult is present)]} #{auxiliary|text|range-tenders} // Range of tenders
				0 {BT-710-LotResult} #{name|BT-710}: ${string(BT-710-LotResult)} ${BT-710-LotResult[BT-710-LotResult/@currencyID != 'unpublished']/@currencyID} // Tender Value Lowest
					{BT-197(BT-710)-LotResult} #{name|BT-197(BT-710)-LotResult}: #{BT-197(BT-710)-LotResult} // Unpublished Justification Code
					{BT-196(BT-710)-LotResult} #{name|BT-196(BT-710)-LotResult}: ${BT-196(BT-710)-LotResult} // Unpublished Justification Description
					{BT-198(BT-710)-LotResult} #{name|BT-198(BT-710)-LotResult}: ${BT-198(BT-710)-LotResult} // Unpublished Access Date
				0 {BT-711-LotResult} #{name|BT-711}: ${string(BT-711-LotResult)} ${BT-711-LotResult[BT-711-LotResult/@currencyID != 'unpublished']/@currencyID} // Tender Value highest
					{BT-197(BT-711)-LotResult} #{name|BT-197(BT-711)-LotResult}: #{BT-197(BT-711)-LotResult} // Unpublished Justification Code
					{BT-196(BT-711)-LotResult} #{name|BT-196(BT-711)-LotResult}: ${BT-196(BT-711)-LotResult} // Unpublished Justification Description
					{BT-198(BT-711)-LotResult} #{name|BT-198(BT-711)-LotResult}: ${BT-198(BT-711)-LotResult} // Unpublished Access Date
			0 {ND-StrategicProcurementLotResult} #{auxiliary|text|statistics-strategic} // Statistics about strategic procurement
				0 {BT-735-LotResult} #{field|name|BT-735-LotResult}: #value  // The type of contract of the procurement procedure within the scope of Clean Vehicles Directive 2009/33/EC.
					0 {BT-723-LotResult} #{field|name|BT-723-LotResult}: #value // The category of vehicle falling within the scope of Directive 2009/33/EC.
						{OPT-155-LotResult} #{OPT-155-LotResult}: ${OPT-156-LotResult} // Vehicle category and number

**29 EFX**

		4 {OPT-322-LotResult[(BT-712(a)-LotResult is present) or (BT-636-LotResult is present) or (BT-760-LotResult is present) or (BT-710-LotResult is present) or (BT-710-LotResult is present) or (BT-735-LotResult is present)]} #{auxiliary|text|statistical-information} // 6.1.4. Statistical information
			0 {OPT-322-LotResult[BT-712(a)-LotResult is present]} #{auxiliary|text|review-requests-summary} // Summary of the review requests the buyer received
				0 {BT-712(a)-LotResult} #{name|BT-712}: ${BT-712(b)-LotResult} // Buyer Review Complainants
					{BT-197(BT-712)-LotResult} #{name|BT-197(BT-712)-LotResult}: #{BT-197(BT-712)-LotResult} // Unpublished Justification Code
					{BT-196(BT-712)-LotResult} #{name|BT-196(BT-712)-LotResult}: ${BT-196(BT-712)-LotResult} // Unpublished Justification Description
					{BT-198(BT-712)-LotResult} #{name|BT-198(BT-712)-LotResult}: ${BT-198(BT-712)-LotResult} // Unpublished Access Date
			0 {OPT-322-LotResult} #{auxiliary|text|review-requests-irregularity-type} // Type of alleged irregularity
			0 {BT-636-LotResult} #{auxiliary|text|review-requests-irregularity-type}: #{BT-636-LotResult} // Type of alleged irregularity
				{BT-197(BT-636)-LotResult} #{name|BT-197(BT-636)-LotResult}: #{BT-197(BT-636)-LotResult} // Unpublished Justification Code
				{BT-196(BT-636)-LotResult} #{name|BT-196(BT-636)-LotResult}: ${BT-196(BT-636)-LotResult} // Unpublished Justification Description
				{BT-198(BT-636)-LotResult} #{name|BT-198(BT-636)-LotResult}: ${BT-198(BT-636)-LotResult} // Unpublished Access Date
				0 {BT-636-LotResult} #{field|name|BT-635-LotResult}: ${BT-635-LotResult} // Buyer Review Requests Count
					{BT-197(BT-636)-LotResult} #{name|BT-197(BT-636)-LotResult}: #{BT-197(BT-636)-LotResult} // Unpublished Justification Code
					{BT-196(BT-636)-LotResult} #{name|BT-196(BT-636)-LotResult}: ${BT-196(BT-636)-LotResult} // Unpublished Justification Description
					{BT-198(BT-636)-LotResult} #{name|BT-198(BT-636)-LotResult}: ${BT-198(BT-636)-LotResult} // Unpublished Access Date
			0 {OPT-322-LotResult[BT-760-LotResult is present]} #{auxiliary|text|received-submissions} // Received tenders or requests to participate
			0 {OPT-322-LotResult[BT-760-LotResult is present]} #{auxiliary|text|received-submissions} // Received tenders or requests to participate
				0 {BT-760-LotResult} #{field|name|BT-760-LotResult}: #{BT-760-LotResult} // Received Submissions Type
					{BT-197(BT-760)-LotResult} #{name|BT-197(BT-760)-LotResult}: #{BT-197(BT-760)-LotResult} // Unpublished Justification Code
					{BT-196(BT-760)-LotResult} #{name|BT-196(BT-760)-LotResult}: ${BT-196(BT-760)-LotResult} // Unpublished Justification Description
					{BT-198(BT-760)-LotResult} #{name|BT-198(BT-760)-LotResult}: ${BT-198(BT-760)-LotResult} // Unpublished Access Date
					0 {BT-760-LotResult} #{field|name|BT-759-LotResult}: ${BT-759-LotResult} // Received Submissions Count
						{BT-197(BT-759)-LotResult} #{name|BT-197(BT-759)-LotResult}: #{BT-197(BT-759)-LotResult} // Unpublished Justification Code
						{BT-196(BT-759)-LotResult} #{name|BT-196(BT-759)-LotResult}: ${BT-196(BT-759)-LotResult} // Unpublished Justification Description
						{BT-198(BT-759)-LotResult} #{name|BT-198(BT-759)-LotResult}: ${BT-198(BT-759)-LotResult} // Unpublished Access Date
			0 {OPT-322-LotResult[(BT-710-LotResult is present) or (BT-710-LotResult is present)]} #{auxiliary|text|range-tenders} // Range of tenders
				0 {BT-710-LotResult} #{name|BT-710}: ${string(BT-710-LotResult)} ${BT-710-LotResult[BT-710-LotResult/@currencyID != 'unpublished']/@currencyID} // Tender Value Lowest
					{BT-197(BT-710)-LotResult} #{name|BT-197(BT-710)-LotResult}: #{BT-197(BT-710)-LotResult} // Unpublished Justification Code
					{BT-196(BT-710)-LotResult} #{name|BT-196(BT-710)-LotResult}: ${BT-196(BT-710)-LotResult} // Unpublished Justification Description
					{BT-198(BT-710)-LotResult} #{name|BT-198(BT-710)-LotResult}: ${BT-198(BT-710)-LotResult} // Unpublished Access Date
				0 {BT-711-LotResult} #{name|BT-711}: ${string(BT-711-LotResult)} ${BT-711-LotResult[BT-711-LotResult/@currencyID != 'unpublished']/@currencyID} // Tender Value highest
					{BT-197(BT-711)-LotResult} #{name|BT-197(BT-711)-LotResult}: #{BT-197(BT-711)-LotResult} // Unpublished Justification Code
					{BT-196(BT-711)-LotResult} #{name|BT-196(BT-711)-LotResult}: ${BT-196(BT-711)-LotResult} // Unpublished Justification Description
					{BT-198(BT-711)-LotResult} #{name|BT-198(BT-711)-LotResult}: ${BT-198(BT-711)-LotResult} // Unpublished Access Date
			0 {ND-StrategicProcurementLotResult} #{auxiliary|text|statistics-strategic} // Statistics about strategic procurement
				0 {BT-735-LotResult} #{field|name|BT-735-LotResult}: #value  // The type of contract of the procurement procedure within the scope of Clean Vehicles Directive 2009/33/EC.
					0 {BT-723-LotResult} #{field|name|BT-723-LotResult}: #value // The category of vehicle falling within the scope of Directive 2009/33/EC.
						{OPT-155-LotResult} #{OPT-155-LotResult}: ${OPT-156-LotResult} // Vehicle category and number

**Comment by pdonohoe** [comment, marked as answer]
Thank you for pointing out this error. BT-759 and BT-760 will be added to the EFX for views 30 to 37 in the next release of the SDK (SDK 1.9), and the fix will be applied to previous SDK versions.
